docs/ops/movement/Unique_10.md

```diff
@@ -32,7 +32,7 @@ The operator can either work in elementwise mode searching for unique values in
 **Inputs**
 
 * **1**: A tensor of type *T* and arbitrary shape. **Required.**
-* **2**: A tensor of type *T_AXIS*. The allowed tensor shape is 1D with a single element or a scalar. **Optional**
+* **2**: A tensor of type *T_AXIS*. The allowed tensor shape is 1D with a single element or a scalar. If provided, this input has to be connected to a Constant. **Optional**
```

Comment:
Can it have a negative value for counting from the end?

Reply:
Yes, to cover the ONNX spec it should handle negative values. I'll update the spec to say that explicitly.